<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterWidget</class>
 <widget class="QWidget" name="ScatterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>500</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>555</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Scatter Plot</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="MplWidget" name="mplWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>100</width>
        <height>500</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="xAxisLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="xlabel">
             <property name="text">
              <string>x axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="xAxisComboBox">
             <property name="toolTip">
              <string>Set which attribute is plotted on the x axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="xLogCheckBox">
             <property name="toolTip">
              <string>Toggle on/off log scaling on the x axis</string>
             </property>
             <property name="text">
              <string>log</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="xFlipCheckBox">
             <property name="toolTip">
              <string>Flip/unflip the order of the x axis</string>
             </property>
             <property name="text">
              <string>flip</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="yAxisLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="ylabel">
             <property name="text">
              <string>y axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="yAxisComboBox">
             <property name="toolTip">
              <string>Set which attribute is plotted on the y axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="yLogCheckBox">
             <property name="toolTip">
              <string>Toggle on/off log scaling on the y axis</string>
             </property>
             <property name="text">
              <string>log</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="yFlipCheckBox">
             <property name="toolTip">
              <string>Flip/unflip the order of the y axis</string>
             </property>
             <property name="text">
              <string>flip</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="LayerTreeWidget" name="layerTree" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerTreeWidget</class>
   <extends>QWidget</extends>
   <header>layertreewidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>xAxisComboBox</tabstop>
  <tabstop>xLogCheckBox</tabstop>
  <tabstop>xFlipCheckBox</tabstop>
  <tabstop>yAxisComboBox</tabstop>
  <tabstop>yLogCheckBox</tabstop>
  <tabstop>yFlipCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
